### PR TITLE
feat(import): add support for customer import

### PR DESCRIPTION
**Pull request by PhilippSpo**
cc/ @emmenko 
- [x] add tests
- [x] add docs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,17 +44,18 @@
     "node": ">= 0.10"
   },
   "dependencies": {
+    "JSONStream": "1.0.7",
     "babel-runtime": "5.8.20",
     "commander": "emmenko/commander.js#patch-help",
+    "ct-customer-import": "0.2.0",
```
Comment (emmenko) [inline package.json]: Why is it called `ct-*` ?

**Comment by emmenko** [comment]
Can you add some unit / feature tests? ;)

File: package.json (hunk 1 above)
Comment (PhilippSpo) [inline package.json]: If I remember correctly we decided together to name it like this

**Comment by PhilippSpo** [comment]
Yes I can add some tests.
btw. Is it really necessary to keep supporting node versions below the current LTS version (which is 4.3.1)?

**Comment by emmenko** [comment]
If it's possible I would still keep support for `0.12`.

File: package.json (hunk 1 above)
Comment (emmenko) [inline package.json]: Did we? I thought we will still publish the npm packages under `sphere-*`

**Comment by PhilippSpo** [comment]
OK then I need to use an old version of `joi` :(

**Comment by PhilippSpo** [comment]
🎉 @emmenko

**Pull request merged**